pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/Account/CreateAccountCpmToken.java

```diff
@@ -47,11 +48,27 @@ protected final String path() {
             if ((scopes & SCOPE_TOPUP) != 0) scopesArray.add("topup");
             put("scopes", scopesArray);
             put("expires_in", expiresIn);
-            put("additional_info", additionalInfo);
+            put("metadata", toJsonString(key1.toMap()));
         }};
     }
 
     public final AccountCpmToken send(String accessToken) throws ProcessingError, BankRequestError {
         return super.send(AccountCpmToken.class, accessToken);
     }
+
+    private String toJsonString(Map<String, Object> map){
```

Comment:
Because in the `Metadata` class which extends from `Parameter`, the method `toMap()` extends from its parent class. So this input type must be `Map<String, objcet>`. And no matter what object is, it will be transformed to string, in here must be json string.

Reply:
I heard from @masatoi that the value must be always a `String`. if a json object is requested to the server it will respond Bad request (invalid parameters)

So I thought it would be better to strict it inside the SDK by typesystem.